### PR TITLE
Docs-1214-About-Calico-Cloud

### PR DESCRIPTION
**Pull request by bmckercher123**
Add new /About content for Calico Cloud

Product Version(s):
3.16 and next

Issue:
https://tigera.atlassian.net/browse/DOCS-1214

Link to docs preview:
https://deploy-preview-617--tigera.netlify.app/calico-cloud/about/
https://deploy-preview-617--tigera.netlify.app/calico-cloud/about/product-comparison

- [x] Deploy preview inspected wherever changes were made
- [x] Build completed successfully
- [x] Tests have passed

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

File: calico-cloud/about/product-comparison.mdx [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+description: Describes Tigera products and provides a feature comparison table.
+---
+
+# Tigera product comparison
+
+## {{prodname}} Open Source
```
Comment (ctauchen) [inline calico-cloud/about/product-comparison.mdx]: These names should be hard-coded. This one reads "Calico Cloud Open Source"

File: calico-cloud/about/about-calico-cloud.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+![calico-cloud](/img/calico/calico-cloud.svg)
+
+Beyond the **Kubernetes security** features that you get from Calico Enterprise and Calico Open Source, {{prodname}} adds these **container security** solutions:
+
+- **Image vulnerability management**
+    Automated image scanning and blocking so you can monitor and assess workloads for new and existing CVEs 24/7. 
+
+- **Runtime container threat defense**
+    Fully-automated protection against known and unknown attacks (network or container-based.
```
Comment (ctauchen) [inline calico-cloud/about/about-calico-cloud.mdx]:
```suggestion
    Fully automated protection against known and unknown attacks (both network and container-based).    
```

File: calico-cloud/about/about-calico-cloud.mdx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- **Image vulnerability management**
+    Automated image scanning and blocking so you can monitor and assess workloads for new and existing CVEs 24/7. 
+
+- **Runtime container threat defense**
+    Fully-automated protection against known and unknown attacks (network or container-based.    
+
+## Best fit
+
+The sweetspot for {{prodname}} is small teams who need to manage the full spectrum of compliance in a web-based console. To jumpstart learning for teams, {{prodname}} provides:
```
Comment (ctauchen) [inline calico-cloud/about/about-calico-cloud.mdx]:
```suggestion
The sweet spot for {{prodname}} is small teams who need to manage the full spectrum of compliance in a web-based console. To jumpstart learning for teams, {{prodname}} provides:
```
I'd just say "The best fit for ..."

File: calico-cloud/about/about-calico-cloud.mdx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Built-in onboarding tutorials in the web console
+- Automatic policy recommendations to make it easy for developers to secure their microservices and applications from day one
+- Hands-on training from Customer Support during the trial period so you can see how to realize your use cases in short order
+- Self-service workshops and training to speed up adoption
+
+## Need more info?
+
+- For specific {{prodname}} features, see [Tigera product comparison](/product-comparison.mdx)
+- To connect your cluster to {{prodname}} in 15 minutes, [start a free trial]()
```
Comment (ctauchen) [inline calico-cloud/about/about-calico-cloud.mdx]: Needs link

File: calico-cloud/about/about-calico-cloud.mdx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+The sweetspot for {{prodname}} is small teams who need to manage the full spectrum of compliance in a web-based console. To jumpstart learning for teams, {{prodname}} provides:
+
+- Built-in onboarding tutorials in the web console
+- Automatic policy recommendations to make it easy for developers to secure their microservices and applications from day one
+- Hands-on training from Customer Support during the trial period so you can see how to realize your use cases in short order
+- Self-service workshops and training to speed up adoption
+
+## Need more info?
+
+- For specific {{prodname}} features, see [Tigera product comparison](/product-comparison.mdx)
```
Comment (ctauchen) [inline calico-cloud/about/about-calico-cloud.mdx]:
```suggestion
- For specific {{prodname}} features, see [Tigera product comparison](product-comparison.mdx)
```

Comment (ctauchen) [reply calico-cloud/about/about-calico-cloud.mdx]: Relative file links

File: calico-cloud/about/about-calico-cloud.mdx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- To connect your cluster to {{prodname}} in 15 minutes, [start a free trial]()
+- [Calico Cloud pricing](https://www.tigera.io/tigera-products/calico-cloud-pricing/)
+- [Connect a cluster to {{prodname}} documentation](/get-started/connect/connect-cluster.mdx)
```
Comment (ctauchen) [inline calico-cloud/about/about-calico-cloud.mdx]: Relative file links. All of these should be something like `(../get-started/bla-bla.mdx)`

**Review by ctauchen — COMMENTED** [review]
In addition the line edits:

* Remove the current index file
* Change links to the current index (from the home page, for example)
* Are there file changes that need redirects? I don't think so.

File: calico-cloud/about/product-comparison.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud/about/product-comparison.mdx]: Fixed. File copied from OSS.

File: calico-cloud/about/about-calico-cloud.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-cloud/about/about-calico-cloud.mdx]: Fixed.

File: calico-cloud/about/about-calico-cloud.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-cloud/about/about-calico-cloud.mdx]: Çhanged as noted.

File: calico-cloud/about/about-calico-cloud.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-cloud/about/about-calico-cloud.mdx]: Fixed.

**Review by ctauchen — COMMENTED** [review]
Needs a few changes to links and a redirect rule to account for moving `https://docs.tigera.io/calico-cloud` to `https://docs.tigera.io/calico-cloud/about`.

But otherwise, all looks good!

**Comment by ctauchen** [comment]
LGTM!

**Pull request merged**